pages/docs/couple-your-code/couple-your-code-direct-access.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Accessing received meshes directly
+permalink: couple-your-code-direct-access.html
+keywords: api, adapter, mapping, meshes
+summary: "This concept is required if you want to access received meshes without a preCICE mapping. The API functions offer access to the data location of the received mesh as well as read and write access to the data itself. "
+---
+
+{% include warning.html content="These API functions have been recently added and are experimental. The API might change during the ongoing development process." %}
+
+This concept is required if you want to access received meshes directly. It might be relevant in case you don't want to use the preCICE own mappings, but rather want to use your own solver for data mapping. As opposed to the usual preCICE mapping, only a single mesh (from the other participant) is now involved in this situation since an 'own' mesh defined by the participant itself is not required any more. In order to re-partition the receiving interface mesh, the participant needs to define the mesh region it wants read data from and write data to. The complete concept on the receiving participant looks as follows:
```

Comment:
```suggestion
This concept is required if you want to access received meshes directly. It might be relevant in case you don't want to use the mapping schemes in preCICE, but rather want to use your own solver for data mapping. As opposed to the usual preCICE mapping, only a single mesh (from the other participant) is now involved in this situation since an 'own' mesh defined by the participant itself is not required any more. In order to re-partition the received mesh, the participant needs to define the mesh region it wants read data from and write data to. The complete concept on the receiving participant looks as follows:
```